QMigrate/io/data.py

```diff
@@ -565,8 +520,8 @@ def get_wa_waveform(self, tr, water_level, pre_filt,
                             water_level=water_level,
                             taper=True,
                             sacsim=True, pitsasim=False, #)  # To replicate remove_response()
-                            paz_simulate=wa_response())
-                # tr.simulate(paz_simulate=get_wa_response())
+                            paz_simulate=util.wa_response())
+                # tr.simulate(paz_simulate=util.wa_response())
```

Comment:
Yeh this is another for me to come back to in future: it would need to be changed slightly to give the option to output cut real waveforms as well as cut wa waveforms. Will clean it up for now.